brainscore/benchmarks/zhu2019.py

```diff
@@ -0,0 +1,211 @@
+        self._stimulus_set = LazyLoad(lambda: load_assembly('extreme_occlusion').stimulus_set)
+        self._visual_degrees = 8
+        self._number_of_trials = 1
+
+        self._metric = Accuracy()
+
+        super(_Zhu2019Accuracy_Engineering, self).__init__(
+            identifier='Zhu2019_extreme_occlusion-accuracy-engineering',
+            parent='Zhu2019',
+            ceiling_func=lambda: Score([1, np.nan], coords={'aggregation': ['center', 'error']}, dims=['aggregation']),
+            bibtex=BIBTEX, version=1)
+
+    def __call__(self, candidate: BrainModel):
+        categories = ["car", "aeroplane", "motorbike", "bicycle", "bus"]
+        candidate.start_task(BrainModel.Task.label, categories)
+        stimulus_set = place_on_screen(self._assembly.stimulus_set, target_visual_degrees=candidate.visual_degrees(),
+                                       source_visual_degrees=self._visual_degrees)
+        labels = candidate.look_at(stimulus_set, number_of_trials=self._number_of_trials)
+        ground_truth = labels["ground_truth"]
+
+        # compare model with stimulus_set (ground_truth)
+        raw_score = self._metric(labels, ground_truth)
+        ceiling = self.ceiling
+        score = raw_score / ceiling.sel(aggregation='center')
+        score.attrs['raw'] = raw_score
+        score.attrs['ceiling'] = ceiling
+        return score
+
+
+"""
+Convert from 19587 trials across 25 subjects to a 500 images x 5 choices assembly.
+This is needed, as not every subject saw every image, and this allows a cross-category comparison
+"""
+
+
+def _human_assembly_categorical_distribution(assembly: DataAssembly, collapse) -> DataAssembly:
+    categories = ["car", "aeroplane", "motorbike", "bicycle", "bus"]
+
+    def categorical(image_responses):
+        frequency = np.array([sum(image_responses.values == category) for category in categories])
+        frequency = frequency / len(image_responses)
+        frequency = type(image_responses)(frequency, coords={'choice': categories}, dims=['choice'])
+        return frequency
+
+    stimulus_coords = ['stimulus_id', 'truth', 'filename', 'image_label', 'ground_truth',
+                       'occlusion_strength', 'image_number', 'word_image']
+    categorical_assembly = assembly.multi_groupby(stimulus_coords).map(categorical)
+    labels = get_choices(categorical_assembly, categories=categories)
+
+    if collapse:
+        categorical_assembly = categorical_assembly.groupby('image_label').sum('presentation')
+        return categorical_assembly
+
+    return labels
+
+
+# takes 5-way response vector and returns category of highest response
+def get_choices(predictions, categories):
+    choices = []
+    for prediction in predictions:
+        choice_index = list(prediction.values).index(max(prediction))
+        choice = categories[choice_index]
+        choices.append(choice)
+    return np.array(choices)
+
+
+# ceiling method:
+class SplitHalvesConsistencyZhu:
```

Comment:
what's different from this to the regular split halves consistency? i.e. why do we need a new class? (comment in code if there's a good reason)